scripts/test.sh

```diff
@@ -48,54 +52,62 @@ function assert() {
 testUserIsAssigned=0
 
 function cleanup() {
-    #always unassign u15 on exit to avoid pollution on the next run. Also ignore all output- this will fail except when the test is exiting early before the second quantization interval
+    #always unassign $userId on exit to avoid pollution on the next run. Also ignore all output- this will fail except when the test is exiting early before the second quantization interval
     if [ $testUserIsAssigned -eq 1 ]
     then
-        echo "Exiting while u15 user should be assigned. Unassigning.."
-        curl -X POST $baseUri/v1alpha/orgs/o1/licenses/smarts -H "Origin: http://smoketest.test" -H "Content-Type: application/json" -H "Authorization:Bearer $token" -d '{"unassign": ["u15"]}'
+        echo "Exiting while $userId user should be assigned. Unassigning.."
+        curl -X POST $baseUri/v1alpha/orgs/$orgId/licenses/smarts -H "Origin: http://smoketest.test" -H "Content-Type: application/json" -H "Authorization:Bearer $token" -d '{"unassign": ["'$userId'"]}'
     fi
 }
 
 login
 trap cleanup EXIT #always run cleanup
 
-msg='Setup: try unassign u15'
-curl --fail -X POST $baseUri/v1alpha/orgs/o1/licenses/smarts -H "Origin: http://smoketest.test" -H "Content-Type: application/json" -H "Authorization:Bearer $token" -d '{"unassign": ["u15"]}' || info "U15 not assigned yet. continuing..."
+msg="Setup: Try unassigning $userId if assigned. should return 400 if not assigned."
+echo $msg
+curl --fail -X POST $baseUri/v1alpha/orgs/$orgId/licenses/smarts -H "Origin: http://smoketest.test" -H "Content-Type: application/json" -H "Authorization:Bearer $token" -d '{"unassign": ["'$userId'"]}' || info "$userId not assigned yet. continuing..."
 
-msg='Granting license to u15 (should succeed)'
-curl --fail -X POST $baseUri/v1alpha/orgs/o1/licenses/smarts -H "Origin: http://smoketest.test" -H "Content-Type: application/json" -H "Authorization:Bearer $token" -d '{"assign": ["u15"]}' || fail "Failed request: $msg"
+msg="Granting license to $userId (should succeed)"
```

Comment:
Assigning seat to user? License is at org level

Reply:
Changed - also in other places where it made sense. @Rajagopalan-Ranganathan PTAL again :)